### DISCUSSION TITLE
Latest version node-canvas

### DISCUSSION THREAD

**Discussion by jack416** [fabricjs/fabric.js#7852] [Q&A]
The latest version of node-canvas is [v2.9.1] , does fabric support？

**Comment by asturur** [comment]
We can update it, i don't think there is any issue with that

**Reply by jack416** [reply]
ok, thank you for your patience.

**Reply by jack416** [reply]
**error in compiling code of  fabric use node-canvas[v2.9.1]** 
_1.fabricjs[v5.1.0]_ 
![2022-04-07_111723](https://user-images.githubusercontent.com/8808197/162113423-df48e602-31a0-4851-a9f1-6dac26dbb2e3.png)
_2.fabricjs[v5.2.1]_ 
![2](https://user-images.githubusercontent.com/8808197/162114821-20610a9e-bc20-47ba-82ec-4bba472ccdb3.png)
